bamboo/bootstrap-integration-tests.sh

```diff
@@ -26,7 +27,7 @@ echo "Locking stack for deployment $DEPLOYMENT"
 
 cd example
 set +e
-
+``
```

Comment:
What is this for?

Reply:
Not sure, didn't show in my editor.   Fixed.